### PR TITLE
Updated tenant spec

### PR DESCRIPTION
**Pull request by hussnain612**
Added clusterResourceWhitelist and namespaceResourceBlacklist in tenant spec for per tenant specifications

### CODE REVIEW

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-67-419131f0`

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-67-ff3280fe`

File: docs/content/sre/tenant-operator/usecases/argocd.md [hunk 1]
```diff
@@ -180,3 +180,38 @@ spec:
     kind: Environment
   ...
 ```
+
+### Override NamespaceResourceBlacklist and ClusterResourceWhitelist per Tenant
+
+Bill now wants a specific tenant to override the `namespaceResourceBlacklist` and `clusterResourceWhitelist` set via Integration Config. Bill will specify these in `argoCD.appProjects` section of Tenant spec. 
+
```
Comment (ShahidHussain-srti) [inline docs/content/sre/tenant-operator/usecases/argocd.md]: modify the sentence/heading to make sure the reader knows that only one of the two can be overriden

Comment (ShahidHussain-srti) [reply docs/content/sre/tenant-operator/usecases/argocd.md]: simplest way is to use "and/or"

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-67-5bf4b146`

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-67-0fd33306`

**Review by ShahidHussain-srti — APPROVED** [review]

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.42`

**Pull request merged**